am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestConstants.java

```diff
@@ -21,6 +21,7 @@
     public static final String SECURITY_CLASSIFICATION = "Public";
     public static final String ACCESSOR_ID = "a";
     public static final String OTHER_ACCESSOR_ID = "b";
+    public static final Set<String> ROLE_NAME = Stream.of("case").collect(toSet());
```

Comment:
Again, ROLE_NAME suggest one role name while it actually holds one element set. Please remove that confusion.

Reply:
👍